Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -34,15 +34,15 @@ class PDHG(Algorithm):
     Parameters
     ----------
     f : Function
-        A convex function with a "simple" proximal method of its conjugate.
+        A convex function with a "simple" proximal method of its conjugate. This function must map from the operator range to the Reals, as :math: `f(Kx)` or :math: `f^{*}(x)` will be the contribution to the total objective. See below for details.
     g : Function
-        A convex function with a "simple" proximal.
+        A convex function with a "simple" proximal. This function must map from the operator domain to the Reals. See below for details.
     operator : LinearOperator
         A Linear Operator.
     sigma : positive :obj:`float`, or `np.ndarray`, `DataContainer`, `BlockDataContainer`, optional, default is 1.0/norm(K) or 1.0/ (tau*norm(K)**2) if tau is provided
-        Step size for the dual problem.
+        Step size for the dual problem. Needs to obey constraints with tau and operator norm to be valid, see below for details.
```

Comment:
```suggestion
        Step size for the dual problem. Needs to obey constraints with tau and operator norm to satisfy convergence guarantees, see below for details.
```
Potentially? Valid is interesting as you get step sizes that don't meet the convergence guarantees but still lead to convergence... and this can vastly speed things up!